app/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,11 @@
     <string name="error_empty_username">Username/Email cannot be empty</string>
     <string name="error_empty_email">Email cannot be empty</string>
     <string name="error_empty_name">Name cannot be empty</string>
+    <string name="error_name_length">The name has to be longer than 1 character and shorter than 31 characters</string>
```

Comment:
once having the limits in resources, you may have to add then using string formating